packages/core/types/index.js

```diff
@@ -47,7 +47,8 @@ export type Target = {
 
 export type Environment = {
   target: Target,
-  browserContext: 'browser' | 'worker' | 'serviceworker'
+  context: 'browser' | 'webWorker' | 'serviceWorker' | 'node' | 'electron',
+  includeNodeModules?: boolean
```

Comment:
How does includeNodeModules get set? Not so relevant to this PR, but I'm curious if this will be done differently once we support externals?

Reply:
Would be set as part of the targets configuration in package.json I guess.